rpm/fs/etc/rc.d/init.d/mad

```diff
@@ -41,13 +63,13 @@ is_running() {
 }
 
 start() {
-    [ -f $config ] || exit 6
+    [ -f $CONFIG_FILE ] || exit 6
```

Comment:
You should use `${CONFIG_FILE}`. Separate issue completely, it would be nice to spruce up some of the shell conventions here and ideally make it posix compliant. There is a utility called shellcheck that does a pretty good lint for this kind of stuff.